etl-pipeline/processes/ingest/gutenberg.py

```diff
@@ -117,16 +105,15 @@ def add_cover(self, gutenberg_record: GutenbergMapping):
             file_type = re.search(r'(\.[a-zA-Z0-9]+)$', cover_data.get('image_path')).group(1)
             cover_path = 'covers/gutenberg/{}{}'.format(gutenberg_id, file_type)
             cover_url = get_stored_file_url(self.file_bucket, cover_path)
+            cover_root = yaml_file.get('url').replace('ebooks', 'files')
+            cover_source_url = f"{cover_root}/{cover_data.get('image_path')}"
```

Comment:
This url 404s most of the time - I can add a HEAD request here to check for file existence. Alternatively, it looks getting the image from the github repo is more reliable, should I swap to doing that?

Reply:
Third alternative - the epub contains a cover which is saved to s3 when the epub is unzipped. We could potentially build something into the record file saver that handles that, but I think that would be a separate ticket that I can create.